src/sst/elements/osseous/examples/vecshiftreg/vecshiftreg.cc

```diff
@@ -0,0 +1,440 @@
+// Copyright 2009-2020 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2020, NTESS
+// All rights reserved.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#include <sst/core/sst_config.h>
+#include <sst/core/event.h>
+#include "VecShiftRegister_O1.h"
+#include "vecshiftreg.h"
```

Comment:
Move line 15 to 13. The corresponding header include should come just after the sst_config include.